docs/book/src/topics/auto-restart-on-rotation.md

```diff
@@ -0,0 +1,120 @@
+# Auto restart of pods when secret is rotated
+
+_This is a suggested implementation which can be used alongside the Secrets Store CSI Driver._
+
+When [auto rotation of secrets](./secret-auto-rotation.md) is enabled, workloads which depend upon secrets will need to either 
+- watch for updates to secrets and reload these in their runtime, or
+- be restarted to pick up the latest secrets when they change
+
+A solution such as [Reloader](https://github.com/stakater/Reloader) could be used to watch for updates to Kubernetes Secrets or ConfigMaps and restart pods when a change is detected. However, if secret values are mounted as volumes in the pods, that solution is not suitable.
```

Comment:
From an official SIG-Auth project, should we be linking out to this in the formal docs?

Reply:
This seems in tone a bit like a blog post for solving a particular problem, rather than a docs site entry. 🤔

Reply:
Per our discussion in the sync meeting, perhaps we keep, but just note that this is not supported or tested.

Reply:
You could see this doc as a continuation from https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/docs/book/src/topics/secret-auto-rotation.md.

I aimed for similar tone.

That doc links to Reloader.

I'm open to suggestions.

My motivation here is that the solution I arrived at and implemented was
- not obvious based on existing resources 
- fairly straightforward given the necessary pointers

Reply:
@benjaminapetersen I have added a note to the top of the doc. If this is not what you have in mind, could you suggest alternative? Thanks